dev-ruby/hashicorp-checkpoint/hashicorp-checkpoint-0.1.5-r1.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+USE_RUBY="ruby25 ruby26 ruby27"
+
+RUBY_FAKEGEM_EXTRADOC="README.md"
+RUBY_FAKEGEM_GEMSPEC="${PN}.gemspec"
+RUBY_FAKEGEM_RECIPE_TEST="rspec3"
+RUBY_FAKEGEM_RECIPE_DOC="none"
+
+inherit ruby-fakegem
+
+DESCRIPTION="Internal HashiCorp service to check version information"
+HOMEPAGE="https://www.hashicorp.com"
+
+LICENSE="MPL-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+ruby_add_bdepend "
+	test? ( dev-ruby/rspec-its )
+"
+
+all_ruby_prepare() {
+	# remove bundler support
+	sed -i '/[Bb]undler/d' Rakefile || die
+	rm Gemfile || die
```

Comment:
If you already remove bundler loading then there is probably no need to remove the Gemfile. Leaving it makes for less clutter in the ebuild and provides a reference file for what dependencies upstream expects.